Dockerfile.dapper

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:19.04
+FROM ubuntu:20.04
```

Comment:
This should be a separate PR

Reply:
will be addressed in https://github.com/rancher/machine/pull/79